internal/hcsoci/hcsdoc_lcow.go

```diff
@@ -25,16 +25,11 @@ func createLCOWSpec(coi *createOptionsInternal) (*specs.Spec, error) {
 	}
 
 	// Linux containers don't care about Windows aspects of the spec except the
-	// network namespace
+	// network namespace and windows devices
 	spec.Windows = nil
-	if coi.Spec.Windows != nil &&
-		coi.Spec.Windows.Network != nil &&
-		coi.Spec.Windows.Network.NetworkNamespace != "" {
-		spec.Windows = &specs.Windows{
-			Network: &specs.WindowsNetwork{
-				NetworkNamespace: coi.Spec.Windows.Network.NetworkNamespace,
-			},
-		}
+	if coi.Spec.Windows != nil {
+		setWindowsNetworkNamespace(coi, spec)
+		spec.Windows.Devices = coi.Spec.Windows.Devices
```

Comment:
Wait I dont get this. Why do we need to forward windows devices to the Linux UVM. Shouldnt this become a Linux device in the spec from the UVM's perspective?

Reply:
Justin and I discussed this offline. We agreed to leave this as is for now.